pudb/theme.py

```diff
@@ -469,57 +469,121 @@ def link(child: str, parent: str):
+            "return value": ("black", "dark cyan"),
+            "focused return label": ("light gray", "dark blue"),
+            "focused return value": ("black", "dark blue"),
             # }}}
             # {{{ stack
-            "current frame name": (
-                add_setting("light gray", "bold"), "black"),
+            "stack": ("white", "black"),
+
+            "frame name": ("white", "black"),
+            "frame class": ("light blue", "black"),
+            "frame location": ("light cyan", "black"),
+
+            "current frame name": (add_setting("white", "bold"), "black"),
+            "current frame class": (add_setting("light blue", "bold"), "black"),
+            "current frame location": (add_setting("light cyan", "bold"), "black"),
+
+            "focused frame name": ("white", "dark blue"),
+            "focused frame class": ("white", "dark blue"),
+            "focused frame location": ("white", "dark blue"),
+
+            "focused current frame name": (
+                add_setting("white", "bold"), "dark blue"),
+            "focused current frame class": (
+                add_setting("white", "bold"), "dark blue"),
+            "focused current frame location": (
+                add_setting("white", "bold"), "dark blue"),
+            # }}}
+            # {{{ breakpoints view
+            "breakpoint": ("white", "black"),
+            "disabled breakpoint": ("dark gray", "black"),
+            "focused breakpoint": ("white", "dark blue"),
+            "focused disabled breakpoint": ("light gray", "dark blue"),
+            "current breakpoint": (add_setting("white", "bold"), "black"),
+            "disabled current breakpoint": (
+                add_setting("dark gray", "bold"), "black"),
+            "focused current breakpoint": (
+                add_setting("white", "bold"), "dark blue"),
+            "focused disabled current breakpoint": (
+                add_setting("light gray", "bold"), "dark blue"),
             # }}}
             # {{{ shell
-            "command line prompt": (
-                add_setting("white", "bold"), "black"),
-            "command line output": ("yellow", "black"),
-            "command line input": (
-                add_setting("white", "bold"), "black"),
+            "command line edit": ("white", "black"),
+            "command line prompt": (add_setting("white", "bold"), "black"),
+
+            "command line output": ("white", "black"),
+            "command line input": ("white", "black"),
+            "command line error": (add_setting("light red", "bold"), "black"),
+
+            "focused command line output": ("white", "dark blue"),
+            "focused command line input": (
+                    "white", "dark blue"),
+            "focused command line error": ("black", "light red"),
+
+            "command line clear button": (add_setting("white", "bold"), "black"),
+            "command line focused button": ("black", "light gray"),
```

Comment:
Unless you don't like green text in the shell, (to unify it with the source view a bit), I vote for simplifying this:
```suggestion
            "command line prompt": (add_setting("white", "bold"), "black"),
            "command line error": (add_setting("light red", "bold"), "black"),
            "command line clear button": (add_setting("white", "bold"), "black"),
```

Reply:
I think I do prefer the white, although I just realized it can have a different color for output, which is probably better. The focused things here aren't needed though.